curvefs/conf/client.conf

```diff
@@ -88,6 +88,10 @@ fuseClient.dCacheLruSize=65536
 fuseClient.enableICacheMetrics=true
 fuseClient.enableDCacheMetrics=true
 fuseClient.cto=true
+# You shoudle enable it when mount one filesystem to multi mountpoints,
+# it gurantee the consistent of file after rename, otherwise you shoudle
+# disable it for performance.
+fuseClient.enableMultiMountPointRename=false
```

Comment:
if you set this config to false, should you prevent current filesystem mount again?

and also, should this config be consistent with fuseClient.cto? which means if cto is true, this config must be true.

Reply:
> if you set this config to false, should you prevent current filesystem mount again?
> 
> and also, should this config be consistent with fuseClient.cto? which means if cto is true, this config must be true.

i had set the default value of `fuseClient.enableMultiMountPointRename` to `true`, if user want to improve 
 the performance in single mount point, should disable it.